modules/menu/static_menu.c

```diff
@@ -1288,6 +1288,13 @@ static int cmd_addheader(struct re_printf *pf, void *arg)
 		return EINVAL;
 	}
 
+	struct mbuf mbe;
+	err = mbuf_printf(&mbe, "%H", uri_header_unescape, &v);
+	if (err == 0) {
+		v.p = (const char *)mbe.buf;
```

Comment:
- `mbe` has to be initialized
- `mbuf_printf` allocates memory in `mbe.buf` and has to be derefed after `ua_add_custom_hdr`
- instead of `err` please use `ret` here